lighthouse-treemap/app/index.html

```diff
@@ -18,7 +18,30 @@
 </head>
 
 <body class="vars">
-  <main class="lh-main lh-main--show-table">
+  <div class="drop_zone"></div>
+
+  <div class="treemap-placeholder">
+    <div class="treemap-placeholder-inner">
+      <div>
+        <img src="images/lighthouse-logo.svg" class="treemap-placeholder-logo" alt="Lighthouse logo">
+      </div>
+      <div>
+        <h1 class="treemap-placeholder__heading">Lighthouse Treemap</h1>
+        <div class="treemap-placeholder__help">To view a treemap: Paste the Lighthouse result json or a Gist URL.<br>
+        You can also drag 'n drop the file or click here to select it.<br><br>
+        To get a fresh report, use the Lighthouse browser extension (<a href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk">Chrome</a> / <a href="https://addons.mozilla.org/en-US/firefox/addon/google-lighthouse/">Firefox</a>)<br>
```

Comment:
The extension bit might be more confusing advice for treemap since they'll end up back over at viewer instead of the treemap and they may or may not have the "view treemap" button.

Reply:
They should always have it after PSI upgrades. does that resolve your concern?

Reply:
> They should always have it after PSI upgrades. does that resolve your concern?

Not particularly. My concern is that if we didn't copy/paste from viewer it feels like we wouldn't have written this that paragraph this way and especially not encourage the extension (the reason we did so for viewer is because the extension is *the* entrypoint for most people using viewer).

The "may or may not have the view treemap button" was also a comment on the general applicability of this page compared to viewer and how appropriate it is to explain that. For example, should we use this space instead to link to a treemap / sourcemaps article on web.dev?